### DISCUSSION TITLE
2023/04/04 Task

### DISCUSSION THREAD

**Discussion by hyunnnn98** [2P3S/scrum-dice#18] [General]
# 🛎 2023/04/03 Task
### 승현

- [x] 화면 url 설계
   - #10
- [x] 컴포넌트 설계
   - #19 
- [ ] [지난주 Task 회의](https://github.com/orgs/2P3S/discussions/6#discussioncomment-5439925)에서 나왔던 디자인 수정

### 재순

- [x] aws iam 설정
- [x] aws elasticache 도입검토
- [x] aws redis / db 연결
- [ ] scrum-dice repo build 관련해서 aws 자동 deploy 하기

### 승형

- [x] 백엔드 클린아키텍처 설계
- [ ] 소켓 url 설계

**Comment by JeongJaeSoon** [comment]
## 추가로 정해야하는 것들
* [x] Issues, Project 사용 방법
  1. Project 에서 task 등록
  2. Draft 에서 Open 으로 전환 (반드시, Assignee, Label 추가)
  3. Issues 에서 Branch 생성
  4. 작업 내용 Barnch 로 Push
  5. Branch 에서 PR 생성하면 모두 자동으로 연결됨
		> 혹시 잘 못 만들어서 연결 안되어있을 경우, 꼭 연결해 둘 것
		> Issues 가 잘 설정되어있다는 전제하에, PR 에는 따로 Assignee, Label 설정은 하지 않음
		>   - 향후 auto labeler, code owner 설정 도입
* [x] Review request rule
	* 기본 룰은 Reviewers 추천 젤 상위 or 아무것도 없는 사람으로 PR 에 설정
	* 추가로 궁금하다, 보고싶다 하면 스스로 참가해도 문제 없음
	* request 받은 사람은 Discord PR에 통지에 👀리액션 붙이기
* [x] Issues & PR 정리
	* 모든 Issues & PR 에는 Issues number 붙일 것
	* exam) [#00] hello world
* [x] PR template 추가
	* 개요
	* PR 에서 한 것
	* 테스트
	* 리뷰어가 봐줬으면 하는 것
* [ ] Wiki 운영
	* WIP

**Comment by JeongJaeSoon** [comment]
##  AWS IAM / Resource 설정
* https://github.com/2P3S/scrum-dice/issues/3
* https://github.com/2P3S/scrum-dice/issues/1
* https://github.com/2P3S/scrum-dice/issues/25
* https://github.com/2P3S/scrum-dice/issues/21

## AWS Redis
* https://github.com/2P3S/scrum-dice/issues/4
